rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridgeManager.java

```diff
@@ -45,13 +45,9 @@
 
     private final Map<String, Xid> transactionXids;
 
-    public static InboundBridgeManager getInstance() {
+    public static synchronized InboundBridgeManager getInstance() {
```

Comment:
its different in that the sync block is not declared on the method. I wasn't advocating a change "I am OK with your change"